sqlalchemy_postgres_autocommit/pytest_fixtures.py

```diff
@@ -24,3 +24,23 @@ def db_session(autocommit_database, db_connection):
     session = autocommit_database.session_factory(bind=db_connection)
     yield session
     session.close()
+
+
```

Comment:
Fixtures should be in 'Add pytest fixtures for transactional tests.' commit not in Add documentation.